TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs

```diff
@@ -11,25 +11,29 @@ namespace TheModernIlluminati.Pages
 {
     public class SearchCountryModel : PageModel
     {
-        public bool searchFinished { get; set; }
+        
         [BindProperty]
         public string CountrySearch { get; set; }
         public string CountryCode { get; set; }
         public void OnGet()
         {
 
         }
-         public void OnPost()
+         public IActionResult OnPost()
```

Comment:
changed the return type to IActionResult so as to enable return Page().

source :  https://www.learnrazorpages.com/razor-pages/action-results